src/xtopcom/xcrypto/src/xckey.cpp

```diff
@@ -174,7 +174,7 @@ namespace top
                 return false;
             }
 
-            auto secp256k1_context_verify = secp256k1_context_create(SECP256K1_CONTEXT_VERIFY);
+            static secp256k1_context* secp256k1_context_verify = secp256k1_context_create(SECP256K1_CONTEXT_VERIFY);
```

Comment:
auto * is correct.

Reply:
std::unique_ptr<secp256k1_context> is the right way, since `secp256k1_context_verify` should be free after use, otherwise memory leaked.